hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/api/AttributeSearchRequest.kt

```diff
@@ -1,17 +1,19 @@
 package uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api
 
+import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.AttributeSearchException
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.Attributes
+import uk.gov.justice.digital.hmpps.prisonersearch.search.services.dto.PaginationRequest
 
+@Schema(description = "A request to search for prisoners by attributes")
 data class AttributeSearchRequest(
-  val queries: List<Query>,
+  @Schema(description = "A query to search for prisoners by attributes")
+  val query: Query,
```

Comment:
Changed from `List<Query>`. A list of top level queries doens't make sense - how are they joined together? There shoudl be only one query.